pykickstart/commands/nvdimm.py

```diff
@@ -0,0 +1,144 @@
+#
+# Radek Vykydal <rvykydal@redhat.com>
+#
+# Copyright 2018 Red Hat, Inc.
+#
+# This copyrighted material is made available to anyone wishing to use, modify,
+# copy, or redistribute it subject to the terms and conditions of the GNU
+# General Public License v.2.  This program is distributed in the hope that it
+# will be useful, but WITHOUT ANY WARRANTY expressed or implied, including the
+# implied warranties of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program; if not, write to the Free Software Foundation, Inc., 51
+# Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.  Any Red Hat
+# trademarks that are incorporated in the source code or documentation are not
+# subject to the GNU General Public License and may only be used or replicated
+# with the express permission of Red Hat, Inc.
+#
+
+import warnings
+from pykickstart.base import BaseData, KickstartCommand
+from pykickstart.errors import KickstartParseError
+from pykickstart.options import KSOptionParser
+from pykickstart.constants import NVDIMM_MODE_SECTOR, NVDIMM_ACTION_RECONFIGURE
+from pykickstart.version import F28
+
+from pykickstart.i18n import _
+
+
+class F28_NvdimmData(BaseData):
+    removedKeywords = BaseData.removedKeywords
+    removedAttrs = BaseData.removedAttrs
+
+    def __init__(self, *args, **kwargs):
+        BaseData.__init__(self, *args, **kwargs)
+        self.action = kwargs.get("action", None)
+        self.namespace = kwargs.get("namespace", "")
+        self.mode = kwargs.get("mode", None)
+        self.sectorsize = kwargs.get("sectorsize", None)
+
+    def __eq__(self, y):
+        if not y:
+            return False
+        return (self.action == y.action and
+                self.namespace == y.namespace)
+
+    def __ne__(self, y):
+        return not self == y
+
+    def _getArgsAsStr(self):
+
```

Comment:
You don't have this blank line anywhere else. Could you please remove it here too?